pkg/release/metrics.go

```diff
@@ -36,3 +46,7 @@ func ObserveRelease(start time.Time, action Action, dryRun, success bool, namesp
 		LabelReleaseName, releaseName,
 	).Observe(time.Since(start).Seconds())
 }
+
+func RecordRelease() {
+	go releasesPerformed.Inc()
```

Comment:
It should be quicker to increment an integer than spinning up a goroutine.  Why do you do this inside `go` ?